driver/tke_driver.go

```diff
@@ -125,6 +130,8 @@ type state struct {
 	ClusterInfo types.ClusterInfo
 	// The total number of worker node
 	NodeCount int64
+
+	CreateClusterRequest *tke.CreateClusterRequest
```

Comment:
Can we only add what needs to be updated here and avoid using TKE defined object directly in the state.
1. To decouple from specific TKE API/SDK version
2. It overlaps the rest of the state structure a lot